src/cdk/tree/tree.ts

```diff
@@ -334,6 +334,8 @@ export class CdkTreeNode<T> implements FocusableOption, OnDestroy {
    */
   @Input() role: 'treeitem' | 'group' = 'treeitem';
 
+  expandable = false;
```

Comment:
Can you prefix this member with an underscore to signal that it's not part of the public API?